pkg/branchcuts/bumper/bumper.go

```diff
@@ -0,0 +1,110 @@
+package bumper
```

Comment:
Are these bumper files used in the second PR?

Reply:
Yes, that's why I have created this abstraction. It will become more obvious when you will be reviewing the second and third PRs.